packages/ontario-design-system-component-library/src/components/ontario-task/ontario-task.scss

```diff
@@ -0,0 +1,57 @@
+@use 'sass:math';
+@use '@ongov/ontario-design-system-global-styles/dist/styles/scss/1-variables/spacing.variables' as spacing;
+@use '@ongov/ontario-design-system-global-styles/dist/styles/scss/1-variables/colours.variables' as colours;
+@use '@ongov/ontario-design-system-global-styles/dist/styles/scss/2-tools/functions/global.functions' as globalFunctions;
+@use '@ongov/ontario-design-system-global-styles/dist/styles/scss/1-variables/breakpoints.variables' as breakpoints;
+
+.ontario-task {
+	border-bottom: 2px solid colours.$ontario-greyscale-20;
+	padding: globalFunctions.px-to-rem(19) spacing.$spacing-0;
+
+	&__content {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+
+		@media screen and (max-width: breakpoints.$small-breakpoint) {
+			display: block;
+		}
```

Comment:
`@media` queries nested within individual selectors makes it hard to manage breakpoints and scale. Can we centralize the media queries, grouped at the end of the file or in a responsive section of the file?